nixnet/types.py

```diff
@@ -388,7 +391,115 @@ def from_raw(cls, frame):
             constants.FrameType.CANFD_DATA: CanFrame,
             constants.FrameType.CANFDBRS_DATA: CanFrame,
             constants.FrameType.CAN_REMOTE: CanFrame,
+            constants.FrameType.LIN_DATA: LinFrame,
+            constants.FrameType.LIN_BUS_ERROR: LinFrame,
+            constants.FrameType.LIN_NO_RESPONSE: LinFrame,
```

Comment:
It's also a special frame: 
[http://zone.ni.com/reference/en-XX/help/372841N-01/nixnet/specialframes/]

Should it be deferred with the rest of the special frame types?